src/views/open-api-docs-view/utils/truncate-hcp-operation-path.ts

```diff
@@ -3,18 +3,47 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+/**
+ * Specific HCP service paths start with the service name,
+ * followed by a date-based version number,
+ * and always include "organization" and "project" parameters.
+ *
+ * We remove all of this prefix from the path for better grouping
+ * of operations within the specific spec.
+ */
+const SPECIFIC_SERVICE_PATTERN =
```

Comment:
In the comments could you get a specific example or explain what this _should_ match. I find it helps when debugging regex in the future to have the human readable explanation. 

Also not sure if you wanted to use that GLOBAL_SERVICE_PATTERN in this via a template literal and `new RegExp(...)`

Reply:
Thanks for the feedback! 💖  🙌 

- **Re: examples**, I added a concrete + a generic example for each regex! Love this as a general rule when defining regexes, will try to do this proactively in the future 🙇‍♂️ 💯 
- **Re: sharing regex parts**, for now have left it a little less "DRY", and have left the version-date matching that's basically all `GLOBAL_SERVICE_PATTERN` does and the same version-date matching in `SPECIFIC_SERVICE_PATTERN` decoupled. I don't see a strong case for abstracting out the underlying regex at this point, but definitely happy to adjust if there are any strong opinions 🙇‍♂️